### PR TITLE
Ga 2011 01

### PR DESCRIPTION
**Pull request by sarang-amondikar**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/billing/resource-manager/Microsoft.Billing/stable/2020-11-01/billingAccount.json [hunk 1]
```diff
@@ -18,774 +18,827 @@
...
-                "x-ms-enum": {
-                  "name": "tradeScreeningState",
-                  "modelAsString": true
+                "accountStatus": {
+                    "description": "The current status of the billing account.",
+                    "type": "string",
+                    "enum": [
+                        "Unknown",
+                        "Active",
+                        "Deleted",
```
Comment (wilcobmsft) [inline specification/billing/resource-manager/Microsoft.Billing/stable/2020-11-01/billingAccount.json]: I thought we were going to remove "Deleted" and merge PendingReview/UnderReview.

File: specification/billing/resource-manager/Microsoft.Billing/stable/2020-11-01/billingAccount.json [hunk 2]
```diff
@@ -18,774 +18,827 @@
...
+                    "type": "number",
+                    "readOnly": true
+                },
+                "currency": {
+                    "description": "The currency in which the charges for the billing profile are billed.",
+                    "type": "string",
+                    "readOnly": true
+                },
+                "endDate": {
+                    "description": "The date when this benefit ends.",
```
Comment (wilcobmsft) [inline specification/billing/resource-manager/Microsoft.Billing/stable/2020-11-01/billingAccount.json]: nit: A spending limit isn't really a benefit?

File: specification/billing/resource-manager/Microsoft.Billing/stable/2020-11-01/billingAccount.json [hunk 3]
```diff
@@ -18,774 +18,827 @@
...
+                    "format": "date-time"
+                },
+                "department": {
+                    "description": "Associated department. By default this is not populated, unless it's specified in $expand.",
+                    "$ref": "#/definitions/Department"
+                }
+            }
+        },
+        "spendingLimitDetails": {
+            "description": "The details of spending limit policy set on billingProfile.",
```
Comment (wilcobmsft) [inline specification/billing/resource-manager/Microsoft.Billing/stable/2020-11-01/billingAccount.json]: "...set on a billing profile."

File: specification/billing/resource-manager/Microsoft.Billing/stable/2020-11-01/billingAccount.json [hunk 4]
```diff
@@ -18,774 +18,827 @@
...
+                        "modelAsString": true
+                    }
+                },
+                "spendingLimitStatus": {
+                    "description": "The status of current spending limit.",
+                    "enum": [
+                        "Active",
+                        "Expired",
+                        "LimitReached",
+                        "LimitRemoved"
```
Comment (wilcobmsft) [inline specification/billing/resource-manager/Microsoft.Billing/stable/2020-11-01/billingAccount.json]: Should it just be "Removed"?

File: specification/billing/resource-manager/Microsoft.Billing/stable/2020-11-01/billingAccount.json (hunk 1 above)
Comment (sarang-amondikar) [reply specification/billing/resource-manager/Microsoft.Billing/stable/2020-11-01/billingAccount.json]: Correct for Deleted, I will remove it, for merging of two states PendingReview/UnderReview, still waiting on response from Trade team

File: specification/billing/resource-manager/Microsoft.Billing/stable/2020-11-01/billingAccount.json (hunk 3 above)
Comment (sarang-amondikar) [reply specification/billing/resource-manager/Microsoft.Billing/stable/2020-11-01/billingAccount.json]: updated

File: specification/billing/resource-manager/Microsoft.Billing/stable/2020-11-01/billingAccount.json (hunk 4 above)
Comment (sarang-amondikar) [reply specification/billing/resource-manager/Microsoft.Billing/stable/2020-11-01/billingAccount.json]: updated

File: specification/billing/resource-manager/Microsoft.Billing/stable/2020-11-01/billingAccount.json (hunk 1 above)
Comment (sarang-amondikar) [reply specification/billing/resource-manager/Microsoft.Billing/stable/2020-11-01/billingAccount.json]: removed Deleted

**Review by wilcobmsft — APPROVED** [review]

File: specification/billing/resource-manager/Microsoft.Billing/stable/2020-11-01/billingProfile.json [hunk 5]
```diff
@@ -176,6 +176,13 @@
           "type": "integer",
           "readOnly": true
         },
+        "invoiceRecipients": {
```
Comment (sarang-amondikar) [inline specification/billing/resource-manager/Microsoft.Billing/stable/2020-11-01/billingProfile.json:179]: how about invoiceRecipientsEmail?

**Pull request merged**